src/constructs/lambda/lambda.test.ts

```diff
@@ -96,6 +96,22 @@ describe("The GuLambdaFunction class", () => {
     expect(SynthUtils.toCloudFormation(stack)).toMatchSnapshot();
   });
 
+  it("should add an alarm if errorPercentageMonitoring is passed in", () => {
+    const stack = simpleGuStackForTesting();
+
+    new GuLambdaFunction(stack, "lambda", {
+      code: { bucket: "bucket1", key: "folder/to/key" },
+      handler: "handler.ts",
+      runtime: Runtime.NODEJS_12_X,
+      errorPercentageMonitoring: {
+        toleratedErrorPercentage: 5,
+        snsTopicName: "test-topic",
+      },
+    });
+
+    expect(stack).toHaveResource("AWS::CloudWatch::Alarm");
```

Comment:
Do we want to test the shape of this resource too?

Reply:
We also have a few tests related to this alarm here: https://github.com/guardian/cdk/blob/main/src/constructs/cloudwatch/lambda-alarms.test.ts. I feel like it's probably better to check specifics about the alarm there?

Reply:
Maybe we can add an inline comment to that effect?